src/integration.test.ts

```diff
@@ -164,6 +164,52 @@ describe(`Integration test for portal ${portal}`, () => {
       },
     };
 
+    it("Should upload and download directories", async () => {
+      const directory = {
+        "i-am-not/file1.jpeg": new File(["foo1"], "i-am-not/file1.jpeg"),
+        "i-am-not/file2.jpeg": new File(["foo2"], "i-am-not/file2.jpeg"),
+        "i-am-not/me-neither/file3.jpeg": new File(["foo3"], "i-am-not/me-neither/file3.jpeg"),
+      };
+      const dirMetadata = {
+        filename: "dirname",
+        length: 12,
+        subfiles: {
+          "i-am-not:file1.jpeg": {
+            contenttype: "image/jpeg",
+            filename: "i-am-not:file1.jpeg",
+            len: 4,
+          },
+          "i-am-not:file2.jpeg": {
+            contenttype: "image/jpeg",
+            filename: "i-am-not:file2.jpeg",
+            len: 4,
+            offset: 4,
+          },
+          "i-am-not:me-neither:file3.jpeg": {
+            contenttype: "image/jpeg",
+            filename: "i-am-not:me-neither:file3.jpeg",
```

Comment:
If I understand correctly, these `:` delimiters are due to some JS oddities and they are to be expected?
The rest of the PR makes sense, this is the only thing that stands out to me.

Reply:
Yep, Karol said (in skynet-prod I think) that they aren't an issue. Thanks for the review!